test/tests.js

```diff
@@ -955,6 +955,29 @@ describe('DOM assertions', function() {
     })
   })
 
+  describe('style', function() {
+    var div = parse('<div style="color: red;">test</div>')
+
+    before(() => {
```

Comment:
Mocha likes regular functions, not arrow fucntions. It binds `this`.

Reply:
@nathanboktae  oh sorry, my bad. I fixed it. Also, it seems like it was rather an issue with the inability to parse ES6 syntax. I don't think bound "this" was the problem  in that case...

Reply:
Yeah this library is ES5 currently.